src/components/Editor/index.js

```diff
@@ -574,12 +582,14 @@ const Editor = React.createClass({
 
     const token = selectedToken.innerText;
     const variables = selectedFrame.scope.bindings.variables;
-    const previewExpression = getExpresionFromToken(
+    const previewExpression = getExpressionFromToken(
       cm,
       sourceText,
       selectedToken
     );
-    if (!variables.hasOwnProperty(token) && !previewExpression) {
+
+    if (!variables.hasOwnProperty(token) && !previewExpression &&
+      !token == "this") {
```

Comment:
just out of curiosity. Why do `!token == "this"` ? I thought it should be `token !== "this"`, or are they the same??